src/renderers/imgui_opengl3.rs

```diff
@@ -1,3 +1,29 @@
-// TODO: Consider making local versions of gl_loader and imgui opengl renderer,
-// but this works for the time being
-pub use {gl_loader, imgui_opengl_renderer};
+use std::ffi::{c_void, CString};
+
+use once_cell::sync::OnceCell;
+use windows::core::PCSTR;
+use windows::Win32::Foundation::{FARPROC, HINSTANCE};
+use windows::Win32::Graphics::OpenGL::wglGetProcAddress;
+use windows::Win32::System::LibraryLoader::{GetProcAddress, LoadLibraryA};
+
+static OPENGL3_LIB: OnceCell<HINSTANCE> = OnceCell::new();
+
+unsafe fn get_opengl3_lib() -> HINSTANCE {
+    let opengl3_cstring = CString::new("opengl32.dll").unwrap();
+
+    LoadLibraryA(PCSTR(opengl3_cstring.as_ptr() as _)).unwrap()
+}
+
+/// # Safety
+///
+/// Help me out lol
```

Comment:
Probably should change to something relevant here before merging too 😛 idk what though